google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -84,6 +84,7 @@
   private static final String EMPTY_BYTE_ARRAY_MD5 = "1B2M2Y8AsgTpgAmY7PhCfg==";
   private static final String EMPTY_BYTE_ARRAY_CRC32C = "AAAAAA==";
   private static final String PATH_DELIMITER = "/";
+  private static final String DEFAULT_STORAGE_HOST = "https://storage.googleapis.com";
```

Comment:
Add a comment stating that signedURLs use the GCS XML API. 
s/DEFAULT_STORAGE_HOST/STORAGE_XML_HOST_NAME/g

Rationale: The GCS library uses the GCS JSON API endpoint but signedUrls use the GCS XML API endpoint. It will also help to clarify that in the code for posterity.